src/components/navBar/demoBasicUsage/index.html

```diff
@@ -0,0 +1,11 @@
+<div ng-controller="AppCtrl" ng-cloak>
+  <md-content class="md-padding">
+    <md-nav-bar selected-link-name="currentLink" nav-bar-aria-label="navigation links">
+      <md-nav-link nav-click="goto('page1')" link-name="page1">Page One</md-nav-link>
+      <md-nav-link nav-click="goto('page2')" link-name="page2">Page Two</md-nav-link>
+      <md-nav-link nav-click="goto('page3')" link-name="page3">Page Three</md-nav-link>
```

Comment:
Actually I don't think I can override, since what I need is not to add additional functionality to ng-click/ui-sref/etc, but to apply the actual directives at a lower level in the dom.